### ISSUE TITLE
Exclude PullRequests from the release notes

### ISSUE THREAD

**Issue by timbussmann** [Particular/GitHubReleaseNotes#32]
In the core repo, we're experimenting with adding milestones to every PR to help interested users to find out which release contains these changes (without digging in the git commits for too long). This becomes very cumbersome as soon as we have an associated issue for a PR which also receives the milestone (because of the release notes) as we have to manually ensure that there are no "duplicates" (and the issue can have a different title as the associated PR) or that only PRs without associated issues have the milestones assigned, which is easy to miss. It seems like one option would be to exclude PRs from the release notes altogether and force the creation of an issue if that PR should be mentioned in the release notes.

Thoughts?

cc @Particular/nservicebus-maintainers

**Comment by danielmarbach** [comment]
Since this would affect other repos as well we need to ping more maintainers. 

Another option would be to not include the PR if the PR mentions an issue with the same milestone (funky logic though)

**Comment by timbussmann** [comment]
> Another option would be to not include the PR if the PR mentions an issue with the same milestone (funky logic though)

yeah that idea came up in yesterdays discussion too but I think that's just going to far.

**Comment by SzymonPobiega** [comment]
Well, there can be a config switch so that the current behavior is default but you can opt in to filter out the PRs.

**Comment by timbussmann** [comment]
> Well, there can be a config switch so that the current behavior is default but you can opt in to filter out the PRs.

good enough imho
